platform/rct/hal/imu_stm32/imu_device_stm32.c

```diff
@@ -39,7 +39,7 @@
 #define IMU_TASK_PRIORITY	(tskIDLE_PRIORITY + 2)
 #define IS_9150_ADDR            0x68
 
-static struct is9150_all_sensor_data sensor_data[2];
+static struct is9150_all_sensor_data sensor_data[2] = {0};
```

Comment:
Bummer.  Just did some extra digging and came to the same conclusion.  Oh well.  I probably would have used some form of label to get it into the data section as opposed to initializing it (just to keep firmware size small) but I don't suppose it much matters in the end.  Good sleuthing for a proper solution that doesn't exist.